common/src/main/java/io/pravega/common/security/TLSProtocolVersion.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.common.security;
+
+import java.util.Arrays;
+
+public class TLSProtocolVersion {
```

Comment:
I'd would have preferred this to be an Enum instead of a class that just encapsulates a String array. That keeps code more readable and maintanable.